src/electionguard_gui/eel_utils.py

```diff
@@ -14,7 +15,9 @@ def utc_to_str(utc_dt: datetime) -> str:
     if not utc_dt:
         return ""
     local = convert_utc_to_local(utc_dt)
-    return local.strftime("%b %#d, %Y %#I:%M %p")
+    if os.name == "nt":
```

Comment:
this also fixes a bug where date time formatting works different on a Mac than on PC